Tools/CISupport/ews-build/steps.py

```diff
@@ -2013,13 +2026,13 @@ class RemoveLabelsFromPullRequest(buildstep.BuildStep, GitHubMixin, AddToLogMixi
         GitHubMixin.REQUEST_MERGE_QUEUE_LABEL,
     ]
 
+    @defer.inlineCallbacks
     def run(self):
         pr_number = self.getProperty('github.number', '')
 
         repository_url = self.getProperty('repository', '')
-        if not self.remove_labels(pr_number, self.LABELS_TO_REMOVE, repository_url=repository_url):
-            return FAILURE
-        return SUCCESS
+        removed_labels = yield self.remove_labels(pr_number, self.LABELS_TO_REMOVE, repository_url=repository_url)
```

Comment:
`removed_labels` name seems as if it's a list of removed labels, while it's a boolean.
Maybe rename it to something like:`success` or `are_labels_removed` or `is_label_removed`